articles/universal-login/new.md

```diff
@@ -0,0 +1,43 @@
+---
+description: New Universal Login Experience
+topics:
+  - login
+  - universal-login
+contentType: index
+toc: true
+---
+# New Universal Login Experience
+
+Auth0's New [Universal Login](/universal-login) experience provides a reimagined login flow, with a fresh UX design and lightweight pages.
+
+The key structural difference with the [Classic Experience](/unniversa-login/classic) is that while the former uses Javascript widgets in all the pages, the New Experience is rendered on the server and does not require Javascript. 
+
+From a functional perspective, there's is a still a feature gap between the New and Classic experience. [Learn more](/universal-login/new-experience-limitations).
+
+In addition of some missing features, in the New Experience some pages has the following functional differences:
+
+## Login
+
+- If you are using Development Keys for Social Providers:
+
+    - Single Sign On and Silent Authentication will work properly, which does not happen in the Classic experience.
+
+    - Users will see a warning in the login page mentioning that the tenant is configured with Development Keys.
+
+- A button will be rendered for each social and enterprise connection. 
+
+- A 'show password' icon will be displayed next to the password field.
+
+## Multi-Factor Authentication
```

Comment:
Also, MFA will be localized. Currently we do not support out of the box i18n for MFA, people need to write their own dicts.